procrastinate/cli.py

```diff
@@ -539,7 +539,11 @@ async def worker_(
     print_stderr(
         f"Launching a worker on {'all queues' if not queues else ', '.join(queues)}"
     )
-    await app.run_worker_async(**kwargs)
+    try:
+        await app.run_worker_async(**kwargs)
+    except asyncio.CancelledError:
+        # prevent the CLI from failing and raising an error when the worker is cancelled
+        pass
```

Comment:
I might be missing something, but I don't think the worker gets cancelled, does it?

I think the only case the worker gets cancelled is if it's run by user code and cancelled by user code. In that case, it behaves correctly. But I believe procrastinate never cancels the worker itself.
(might have missed something)

Reply:
You are correct. This is left over from the prior attempt at using cancellation instead of a stop event.